clang-tools-extra/docs/clang-tidy/GlobalOptions.rst

```diff
@@ -0,0 +1,21 @@
+==============
+Global Options
+==============
+
+Some options apply to multiple checks. This page lists all the available
+globally options.
+
+.. option:: IncludeStyle
+
+   A string specifying which include-style is used, `llvm` or `google`. Default
+   is `llvm`.
+
+.. option:: IgnoreMacros
+
+   If set to `true`, the check will not give warnings inside macros. Default
+   is `true`.
+
+.. option:: StrictMode
```

Comment:
This option does not have any meaning in this context. Therefore I believe it should be kept local.